server/controller/root.py

```diff
@@ -104,6 +113,60 @@ def historical():
         'page': 'historical',
     }
 
+# Placeholder
+@get('/paste')
+@kajiki_view('paste')
+@authenticated()
+def paste():
+    if not controller.user_management_interface.is_user_logged_in(request):
+        redirect('/login_page')
+    api_key = controller.user_management_interface.get_api_key_for_user(request)
+    return {'page': 'paste',
```

Comment:
Format the return like:

    return {
        'page': 'upload',
        'raw_logs': '',
    }

Note the name change of the page (you don't have to call it exactly that).

Also note the space between the `:` and `'` on the raw logs line.